docs/reference/method/MongoDBDatabase-aggregate.txt

```diff
@@ -0,0 +1,71 @@
+==============================
+MongoDB\\Database::aggregate()
+==============================
+
+.. versionadded:: 1.5
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Definition
+----------
+
+.. phpmethod:: MongoDB\\Database::aggregate()
+
+   Executes an :manual:`aggregation framework pipeline
+   </core/aggregation-pipeline>` operation on the collection.
+
+   .. code-block:: php
+
+      function aggregate(array $pipeline, array $options = []): Traversable
+
+   This method has the following parameters:
+
+   .. include:: /includes/apiargs/MongoDBDatabase-method-aggregate-param.rst
+
+   The ``$options`` parameter supports the following options:
+
+   .. include:: /includes/apiargs/MongoDBDatabase-method-aggregate-option.rst
+
+Return Values
+-------------
+
+A :php:`MongoDB\\Driver\\Cursor <class.mongodb-driver-cursor>` or
+:php:`ArrayIterator <arrayiterator>` object. In both cases, the return value
+will be :php:`Traversable <traversable>`.
```

Comment:
I was going to suggest double-checking to see if `explain` is compatible with any of the [DB-level stages](https://docs.mongodb.com/manual/reference/operator/aggregation-pipeline/#db-aggregate-stages), but even if none of those are supported it's possible something is introduced in the future that does work with `explain` and might result in an ArrayIterator returned. I think we can leave this as-is. It's still accurate, even without the verbose "Behavior" section left in.